install_config/topics/ocp_osp_sdn.adoc

```diff
@@ -36,3 +36,9 @@ To enable Kuryr SDN, your environment must meet the following requirements:
 * Neutron Trunk ports extension enabled
 * If ML2/OVS Neutron driver is used the OpenvSwitch firewall driver must be
 used, instead of the ovs-hybrid one.
+
+[IMPORTANT]
+====
+To use Kuryr with {rhosp} 13z13 you need to use version v3.11.306 or higher of
+Kuryr container images.
+====
```

Comment:
```suggestion
[IMPORTANT]
====
To use Kuryr with {rh-openstack-first} 13.0.13, the Kuryr container images must be version v3.11.306 or higher.
====
```
The MM.zz pattern doesn't seem like it's really used in OCP or OSP documentation. Where'd you see it originally, and do you agree with my substitution?

Reply:
You're right, I now see that we only use 13z13 internally and according to https://bugzilla.redhat.com/show_bug.cgi?id=1873179 13.0.13 is the correct notation.